pages/directory-score.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { StyleSheet } from 'react-native';
+
+import { H1, H2, H3, P, Headline, A, colors } from '../common/styleguide';
+import ContentContainer from '../components/ContentContainer';
+
+// type Props = {};
+
+function DirectoryScore(props) {
+  return (
+    <ContentContainer style={styles.container}>
+      <H1 style={styles.header}>Directory Score</H1>
+      <P style={styles.paragraph}>
+        The Directory Score is the combination of multiple factors that relate to the quality of a
+        library. A library can earn value by exhibiting "good behavior criteria" and can lose value
+        by exhibiting "bad behavior criteria". These criteria and their corresponding weights are
+        detailed below. You can see the code this directory uses to create Directory Scores{' '}
+        <A href="https://github.com/react-native-directory/website/blob/master/scripts/calculate-score.js">
+          here
+        </A>
+        .
+      </P>
+      <P style={[styles.paragraph, styles.callout]}>
+        Directory Scores are subjective and are based on data that's readily available on GitHub.
+        It's not a perfect score and may not reflect quality for your specific needs. When
+        evaluating libraries to include in your project, review the library yourself to determine if
+        it's a good fit.
+      </P>
+      <H2 style={styles.subHeader}>Scoring criteria</H2>
+      <P style={styles.paragraph}>
+        The following criteria are used to calculate a library's Directory Score.
+      </P>
+      <Headline style={styles.criterion}>Very popular (+40)</Headline>
+      <P style={styles.paragraph}>
+        Libraries with a popularity score of over 10,000 meet this criterion. Popularity is measured
+        by weighting and combining subscribers, forks, stars, and download counts.
+      </P>
+      <Headline style={styles.criterion}>Popular (+20)</Headline>
```

Comment:
It looks like popular only gets a score of `10` https://github.com/react-native-directory/website/blob/master/scripts/calculate-score.js#L17

```suggestion
      <Headline style={styles.criterion}>Popular (+10)</Headline>
```